src/Client.php

```diff
@@ -243,6 +243,14 @@ protected function getExceptionMap()
     }
 
     /**
+     * @return LoggerInterface
+     */
+    public function getLogger()
```

Comment:
I feel that's actually something worth doing either way.
For instance, this makes what I wanted to do in https://github.com/contentful/contentful.php/pull/162 much easier, as you don't need to keep the logger around because you just can retrieve it from the client.

The fact that it was added to cover a specific need here doesn't mean it'd not be useful overall, on the contrary...